### DISCUSSION TITLE
Problem with Config file?

### DISCUSSION THREAD

**Discussion by laymara-xs** [HOPE-UIB-BIO/FOSSILPOL-issues#26] [Q&A]
It literally just has a heading and then says just this:

# Configuration script with the variables that should be consistent throughout
#   the whole repo. It loads packages, defines important variables,
#   authorises the user, and saves the config file.
# Points that require the userb

It finishes with an apparently incomplete phrase. I had cloned the repo but I also downloaded again the separate file to check (thought I might have erased things) and it looks the same. What am I missing?

**Comment by OndrejMottl** [comment]
Hi @laymara-xs, 

That is an interesting error. The `R/00_Config_file.R` looks OK on the GitHub (see [here](https://github.com/HOPE-UIB-BIO/FOSSILPOL-workflow/blob/main/R/00_Config_file.R)), or have you downloaded some other file?

Is it just about this one file or are other scripts shortened or can you see this in other Scripts?

I can see that in the [version on your GitHub](https://github.com/laymara-xs/FOSSILPOL/blob/main/R/00_Config_file.R), the file is fine as well. 

There are 2 solutions:

1. clone the repo fresh again
2. Copy the script from GitHub directly ([link to raw](https://raw.githubusercontent.com/HOPE-UIB-BIO/FOSSILPOL-workflow/main/R/00_Config_file.R))

Let me know how else can I help.

**Reply by OndrejMottl** [reply]
@laymara-xs any progress? Can I close this discussion?
